internal/test/k8s_helpers.go

```diff
@@ -0,0 +1,78 @@
+package test
+
```

Comment:
This file is probably already in `master` by the time I publish the review (as well as `controller_test.go`)